### PR TITLE
Audit fix

### PR DESCRIPTION
**Pull request by vladshablinsky**
Fix problems in formulae.

### CODE REVIEW

**Comment by mikemcquaid** [comment]
@vladshablinsky If you run `brew audit --strict` you'll see even more warnings. Could you try that and fix them too? No worries if not and I'm happy to merge this as-is.

**Comment by vladshablinsky** [comment]
Yes I'll fix them too.

**Comment by vladshablinsky** [comment]
@mikemcquaid I fixed all but `test do` blocks.

File: Library/Formula/sbuild.rb [hunk 1]
```diff
@@ -1,13 +1,10 @@
-require 'formula'
-
 class Sbuild < Formula
   homepage 'http://sbuild.org'
   url 'http://sbuild.org/uploads/sbuild/0.7.6/sbuild-0.7.6-dist.zip'
-  sha1 'd3c815eea2db0347c2b8a319d52c32a50b03d803'
-
+  sha256 '64e17df4db26980170e5cf616d015924da01f232c5207f2bdc7ba10e7d976e3d'
```
Comment (mikemcquaid) [inline Library/Formula/sbuild.rb]: Can you add a newline under here, thanks.

**Comment by mikemcquaid** [comment]
@vladshablinsky Try running `brew audit --strict sbuild yaml-cpp yajl unp` and fixing those warnings too (e.g the double-quotes). Thanks!

**Comment by vladshablinsky** [comment]
@vladshablinsky No offenses detected for now.

File: Library/Formula/sbuild.rb [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
-  sha1 'd3c815eea2db0347c2b8a319d52c32a50b03d803'
+  homepage "http://sbuild.org"
+  url "http://sbuild.org/uploads/sbuild/0.7.6/sbuild-0.7.6-dist.zip"
+  sha256 "64e17df4db26980170e5cf616d015924da01f232c5207f2bdc7ba10e7d976e3d"
 
   def install
-    libexec.install Dir['*']
-    system "chmod +x #{libexec}/bin/sbuild"
+    libexec.install Dir["*"]
+    chmod 0755, "#{libexec}/bin/sbuild"
```
Comment (DomT4) [inline Library/Formula/sbuild.rb]: You should be able to change this to ` chmod 0755, libexec/"bin/sbuild" `. It's just a style thing really, though.

**Comment by DomT4** [comment]
This is awesome :+1:

**Comment by asparagui** [comment]
Nice work!

**Pull request closed without merging**